cmd/common.go

```diff
@@ -59,5 +64,22 @@ func processCommonEvalFlags(flags *flag.FlagSet, args *[]string) error {
 	if exec != "" {
 		*args = append(*args, toDataURL(exec))
 	}
+
+	overlay, err := flags.GetString(flagOverlay)
+	if err != nil {
+		return err
+	}
+	if overlay != "" {
+		alpha := viper.GetBool(flagAlpha)
+		if !alpha {
+			return fmt.Errorf("--overlay is an alpha feature please use --alpha")
```

Comment:
Minor nitpick:

Since this is `Errorf`, but with no format directive could we use the flag variables in their relevant places?

Granted, the more I think about that the uglier it actually looks, just a thought either way :laughing:

Reply:
yeah, it's a tradeoff; it's not about ugliness, it's more about code searchability vs ensured consistency. I'm accepting your suggestion since it's indeed possible that alpha flags like this get renamed before stabilizing.